script.js

```diff
@@ -1,6 +1,129 @@
+  },
+  {
+    Name: 'Profesional Art Printing Data',
+    Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    image: 'Images/abvout.png',
+    Tech: ['Html', 'CSS', 'Bootstrap', 'Ruby'],
+    button: 'See project',
+  },
+  {
+    Name: 'Profesional Art Printing Data ',
+    Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    image: 'Images/abvout.png',
+    Tech: ['Html', 'CSS', 'Bootstrap', 'Ruby'],
+    button: 'See project',
+  },
+  {
+    Name: 'Profesional Art Printing Data',
+    Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    image: 'Images/abvout.png',
+    Tech: ['Html', 'CSS', 'Bootstrap', 'Ruby'],
+    button: 'See project',
+  },
+  {
+    Name: 'Profesional Art Printing Data',
+    Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    image: 'Images/abvout.png',
+    Tech: ['Html', 'CSS', 'Bootstrap', 'Ruby'],
+    button: 'See project',
+  },
+];
+
+projectCards.forEach((card) => {
+  const content = `
+  <div>
+  <h3>${card.Name}</h3>
+
+  <p>${card.Description}</p>
+
+  <ul>
+      <li>${card.Tech[0]}</li>
+      <li>${card.Tech[1]}</li>
+      <li>${card.Tech[2]}</li>
+  </ul>
+  <button id="sampleproject" type="button">${card.button}</button>
+</div>
+  `;
+  workCards.innerHTML += content;
+});
+
+const sampleStep = [
+  {
+    title: 'Multi Post Stories',
+    Description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    Image: 'Images/Portfolio.png',
+  },
+];
+const btnProject = document.querySelectorAll('#sampleproject, .recentBtn');
+
+btnProject.forEach((item) => {
+  item.addEventListener('click', () => {
+    const main = document.createElement('div');
+
+    const popup = document.createElement('div');
+    popup.className = 'popup-container';
+
+    popup.innerHTML = `
+  <div  class="element-class">
+  <div class="titles">
+      <h1> ${sampleStep[0].title}</h1>
+      <i  class="bi bi-x closeBtn"></i>
+</div>
+
+<ul>
+     <li>html</li> 
+     
+     <li>Bootstrap</li>
+     <li>Ruby on rails</li>  
+</ul>
+<div class="parapragh-section">
+<div class="image-class">
+     <img class="img-class" src="${sampleStep[0].Image}">
+</div>
+ <div class="class-container">
+<p>
+${sampleStep[0].Description}
+</p> 
+
+<div class="button-container">
+     <button type="submit">See Live</button>
+     
+     <button type="submit">See Source</button>
+</div>
```

Comment:
- [ ] Please add the `live` and `GitHub` icons inside the project buttons.

### Your design
![image](https://user-images.githubusercontent.com/83803414/189079373-bfab8dbe-2ecf-42b5-a439-eac8a2e61d16.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/189079413-7350451f-d150-46e1-aca5-0f2741fdd2d3.png)